### PR TITLE
Bridge grammartec from Nautilus to libafl

### PR DESCRIPTION
**Pull request by andreafioraldi**

### CODE REVIEW

**Comment by andreafioraldi** [comment]
@domenukk there is a way to stop the CI on old commits when I push a new one?

**Comment by domenukk** [comment]
> @domenukk there is a way to stop the CI on old commits when I push a new one?

don't think so, but you can manually cancel a ci run if you want, in github's UI

File: libafl_concolic/symcc_libafl/src/lib.rs [hunk 1]
```diff
@@ -22,9 +22,10 @@ mod clone {
     /// Any errors will trigger a panic.
     pub fn clone_symcc_at_version(path: &Path, url: &str, commit: &str) {
         assert!(
-            !which("git").is_err(),
```
Comment (andreafioraldi) [inline libafl_concolic/symcc_libafl/src/lib.rs]: lol this was the clippy suggestion

**Comment by domenukk** [comment]
New toolchain with new clippy version?

**Comment by domenukk** [comment]
We were at <1h CI time, now we're > 1.5h, what changed?

**Pull request merged**